yarn.lock

```diff
@@ -3345,7 +3345,7 @@
 
 "@jridgewell/gen-mapping@^0.3.0":
   version "0.3.2"
-  resolved "https://registry.yarnpkg.com/@jridgewell/gen-mapping/-/gen-mapping-0.3.2.tgz#c1aedc61e853f2bb9f5dfe6d4442d3b565b253b9"
```

Comment:
Ensure that you do not modify anything from this file. You can discard these changes from your pr as they will interfere with other collaborators on the repo.
when you're done, you can tag me to confirm that you've made the changes

Reply:
Ok. Thanks. Hope I haven't made changes to the yarn.lock already?

Reply:
No you haven't. The changes only reflect if your or is merged